src/config.yml

```diff
@@ -18,7 +18,6 @@ params:
             - "js/jquery.cookie.js"
             - "js/anime.min.js"
             - "js/platform.min.js"
-            - "js/pts5.js"
```

Comment:
If we are no longer using `js/pts5.js`, can it be removed from the repo?

I notice we also have `pts1.js`, `pts2.js` etc. Can we remove those as well?

Reply:
Yeah those are old leftovers, probably best to do a separate PR to clean them out.

Reply:
Removed all these pts libs as they've long been replaced with videos.